docs/tokens/nft-tokens.md

```diff
@@ -596,13 +596,21 @@ TransferTransaction {
     Value: 0
     GasLimit: 1000000 + length of Data field in bytes * 1500
     Data: "ESDTNFTTransfer" +
-          "@" + <token identifier in hexadecimal encoding> +
-          "@" + <the NFT nonce in hexadecimal encoding> +
+          "@" + <collection identifier in hexadecimal encoding> +
+          "@" + <the NFT nonce> +
           "@" + <quantity to transfer in hexadecimal encoding> +
           "@" + <destination address in hexadecimal encoding>
 }
 ```
 
+:::tip
+Here is an example of an NFT identifier: `ABC-1a9c7d-05dc`
+
+The collection identifier is `ABC-1a9c7d` and the NFT nonce is `05dc`
```

Comment:
also add: ". Note that the `05dc` is hexadecimal encoded. It represents decimal 1500"